src/styles/index.css

```
@@ -19,104 +19,106 @@
-    --color-muted: theme(--color-zinc-800);
-    --color-muted-foreground: theme(--color-zinc-400);
-    --color-accent: theme(--color-purple-950);
-    --color-accent-foreground: theme(--color-purple-50);
-    --color-destructive: theme(--color-red-900);
-    --color-destructive-foreground: theme(--color-red-50);
-    --color-border: theme(--color-zinc-800);
-    --color-input: theme(--color-zinc-800);
-    --color-ring: theme(--color-purple-400);
+    /* Elphaba — Dark Mode: Deeper moss green background */
+    --color-background: #04120b;
+    --color-foreground: theme(--color-emerald-50);
```

Biome (2.1.2)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)